frontend/packages/ceph-storage-plugin/src/components/namespace-store/namespace-store-form.tsx

```diff
@@ -176,6 +184,45 @@ const NamespaceStoreForm: React.FC<NamespaceStoreFormProps> = withHandlePromise<
           dispatch={providerDataDispatch}
         />
       )}
+      {provider === BC_PROVIDERS.FILESYSTEM && (
+        /* eslint-disable jsx-a11y/label-has-associated-control */
+        <>
+          <FormGroup
+            label={t('ceph-storage-plugin~Persistent volume claim')}
+            fieldId="pvc-name"
+            className="nb-endpoints-form-entry"
+            isRequired
+          >
+            <PVCDropdown
+              dataTest="pvc-dropdown"
+              namespace={namespace}
```

Comment:
```suggestion
              id="pvc-name"
              namespace={namespace}
```

Reply:
are you asking to remove the namespace ?

Reply:
nope... to add `id` corresponding to `fieldId` added in the `FormGroup`

Reply:
It is there ? :O